tests/unit/channel_test.py

```diff
@@ -33,19 +33,21 @@ def ssh_channel(ssh_client_session):
         chan.close()
 
 
-@pytest.mark.xfail(
-    reason='This test causes SEGFAULT, flakily. '
-    'Ref: https://github.com/ansible/pylibssh/issues/57',
-    strict=False,
-)
+def exec_second_command(ssh_channel):
+    """Call exec_command() from different context in the call stack."""
```

Comment:
Calling from a different context is a part of the motivation. Within this function, there's nothing that changes the context. You're probably referring to the calling test but this is not something that would go into the docstring because it's not what the function does, it's what that test does, probably.

Reply:
```suggestion
    """Retrun ``exec_command()`` stdout as a text string."""
```

Reply:
I am not sure if this is more accurate as the function does not return anything -- just runs the command and checks the output.

Reply:
Ah, right, I made another suggestion now.